unittests/test_policies.py

```diff
@@ -132,6 +133,9 @@ def test_system_exit_within_test(self):
         stats = self.runner.stats
         self.assertEqual(1, stats.num_failures())
 
+    # Retries tests are executed in a different runtime as they modify
+    # the global rt.runtime().current_run what makes subsequent tests fail
```

Comment:
This comment is not very accurate. You should say that the policy tests need a freshly initialized runtime, that's why we need to switch before every test.